exampleCourse/infoCourse.json

```diff
@@ -20,11 +20,13 @@
     "tags": [
         {"name": "mwest", "color": "gray1", "description": "Question originally written by mwest"},
         {"name": "tbretl", "color": "blue1", "description": "Question originally written by tbretl"},
+        {"name": "mfsilva", "color": "blue1", "description": "Question originally written by mfsilva"},
         {"name": "balamut2", "color": "orange1", "description": "Question originally written by balamut2"},
         {"name": "tpl101", "color": "gray1", "description": "Question originally written for the XC 101 example course"},
         {"name": "fa17", "color": "gray1", "description": "Question written in Fall 2017"},
         {"name": "sp18", "color": "gray1", "description": "Question written in Spring 2018"},
         {"name": "v2", "color": "red1", "description": "v2 generation question"},
-        {"name": "v3", "color": "green1", "description": "v3 generation question (freeform)"}
+        {"name": "v3", "color": "green1", "description": "v3 generation question (freeform)"},
+        {"name": "pearl", "color": "yellow3", "description": "Question originally written by pearl"}
```

Comment:
Maybe move this up to follow the other named users, instead of following v2/v3?

Reply:
Are you suggesting we shouldn't partially grade an element that has nothing submitted? Do we have scenarios where 0 correct answers are included (i.e. they must check something)?

Reply:
Do we want to mark as "invalid" when the submitted answer is left blank (and so it is not graded)? In that case, instructors will always have to make at least one of the answers correct, which I think is reasonable. 

Should I go ahead and make this implementation? Or any other suggestions?

Reply:
@mfsilva22 I think we do want to mark the answer as invalid. I say this because the logic was never implemented to check for empty submissions, c.f.

https://github.com/PrairieLearn/PrairieLearn/blob/12ca790b3c93adadb0cdfdc43962d96ca54e2b37/elements/pl-checkbox/pl-checkbox.py#L216-L218